src/game.py

```diff
@@ -1,3 +1,13 @@
+from enum import Enum
+
+
+class Move(Enum):
+    TOP = 1
```

Comment:
You don't have to use numbers here. Use strings `TOP = 'TOP'`, it's easier to debug